pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -549,7 +548,7 @@ func (ds *dockerService) ListPodSandbox(_ context.Context, r *runtimeapi.ListPod
 		c := containers[i]
 		converted, err := containerToRuntimeAPISandbox(&c)
 		if err != nil {
-			klog.V(4).Infof("Unable to convert docker to runtime API sandbox %+v: %v", c, err)
+			klog.V(4).InfoS("Unable to convert docker to runtime API sandbox", "container", c, "err", err)
```

Comment:
I think this is the only place that has access to `container` object and it's not a Kubernetes object so I would like to avoid using it for now. Please let me know it this will work.
```suggestion
			klog.V(4).InfoS("Unable to convert docker to runtime API sandbox", "containerName", c.Name, "err", err)
```